cluster-autoscaler/Makefile

```diff
@@ -21,7 +21,7 @@ test-unit: clean deps build
 	$(ENVVAR) godep go test --test.short -race ./... $(FLAGS)
 
 test-e2e:
-	$(ENVVAR) godep go run ./test/e2e/*.go -alsologtostderr
+	cd test && go run ./e2e/*.go -alsologtostderr
```

Comment:
can you either squash into the previews commit modifying make file or having all make changes into a single atomic commit?

Reply:
Please merge the first and the last commit into one.